team_production_system/models/availability.py

```diff
@@ -0,0 +1,19 @@
+from .mentor import Mentor
+from django.db.models.constraints import UniqueConstraint
+from django.db import models
+
```

Comment:
Pre-existing linting error: 
Needs one more blank line between imports and class (comment descriptions do not apply)